server/src/main/java/org/apache/iotdb/db/auth/AuthorizerManager.java

```diff
@@ -42,6 +57,19 @@ public class AuthorizerManager implements IAuthorizer {
 
   private IAuthorizer iAuthorizer;
   private ReentrantReadWriteLock snapshotLock;
+  private TPermissionInfoResp tPermissionInfoResp;
+
+  private LoadingCache<String, User> userCache =
+      Caffeine.newBuilder()
+          .maximumSize(100)
+          .expireAfterAccess(30, TimeUnit.MINUTES)
+          .build(this::cacheUser);
+
+  private LoadingCache<String, Role> roleCache =
+      Caffeine.newBuilder()
+          .maximumSize(100)
+          .expireAfterAccess(30, TimeUnit.MINUTES)
+          .build(this::cacheRole);
```

Comment:
fix~

Reply:
Actually I have another opinion here. That is, we can put the magic numbers as constant instead of put them all into the configuration to avoid confusing the users...

This is just an idea, we can talk about it later